### PR TITLE
Added recipe for Pandemy version 1.0.0

### PR DESCRIPTION
**Pull request by antonlydell**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by antonlydell** [comment]
Dear conda-forge team,

I need some advice on my recipe.

Pandemy has the following requirements to run:
- python >=3.8
- pandas >=1.2.0
- sqlalchemy >=1.4.0

It further needs setuptools >=42 to build.

I get the linting issue: "Non noarch packages should have python requirement without any version constraints."
I want to set the run requirement to python >= 3.8. How do I do this?

Both pandas and SQLAlchemy are non noarch packages. Should I add noarch: python to the build section in meta.yaml?
Pandemy can run on any platform as long as pandas and SQLAlchemy are installed.

Thanks and Happy Easter
@conda-forge-admin please ping conda-forge/python

**Comment by antonlydell** [comment]
I hereby confirm that I (antonlydell) am willing to be become a maintainer of the recipe for Pandemy.

**Comment by antonlydell** [comment]
Dear conda-forge team,

The recipe for Pandemy has passed the checks and is ready for review and merge.
@conda-forge/staged-recipes

Wishing you a Happy Easter,

Best regards
Anton Lydell

File: recipes/pandemy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "Pandemy" %}
+{% set version = "1.0.0" %}
```
Comment (beenje) [inline recipes/pandemy/meta.yaml:2]: I noticed that version 1.1.0 is already available. Do you really want to publish 1.0.0 as conda package? Or should you bump to 1.1.0 this recipe?

File: recipes/pandemy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: def2257d3044e3249ddde28b916db20ad4d9b751271beae5bea38a40c525d863
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (beenje) [inline recipes/pandemy/meta.yaml:15]: The standard script command for Python packages is `script: {{ PYTHON }} -m pip install . -vv`.
`--no-deps` isn't needed as it is added automatically by conda-build.

No need to change. This is just a comment.

**Comment by beenje** [comment]
The recipe looks good to me. You found the answers to your questions yourself :-)
I just have one comment about the version.

File: recipes/pandemy/meta.yaml (hunk 1 above)
Comment (antonlydell) [reply recipes/pandemy/meta.yaml:2]: Hi beenje!

Yes I want to publish version 1.0.0 before version 1.1.0. I then want to go through the process of releasing a new version of the recipe (version 1.1.0) as practice exercise.

File: recipes/pandemy/meta.yaml (hunk 2 above)
Comment (antonlydell) [reply recipes/pandemy/meta.yaml:15]: Thanks for the feedback I will remove --no-deps in version 1.1.0.

**Comment by antonlydell** [comment]
Sounds good!
Yes, after some more research about noarch: python I found that it made the most sense for Pandemy.
I want to publish version 1.0.0 before version 1.1.0 and then go through the process of releasing a new version of the recipe (version 1.1.0) as practice exercise.

**Comment by antonlydell** [comment]
Dear beenje,

I have updated the recipe for Pandemy to version 1.1.0 (the most recent version) and the checks have passed.
I have also removed --no-deps from the pip install script as you suggested before.
Please review the changes.

Best regards
Anton Lydell
@conda-forge-admin, please ping team

**Pull request merged**